components/kyma-environment-broker/internal/provisioner/graphqlizer.go

```diff
@@ -139,6 +141,11 @@ func (g *Graphqlizer) UpgradeClusterConfigToGraphQL(in gqlschema.UpgradeClusterI
 }
 
 func (g *Graphqlizer) KymaConfigToGraphQL(in gqlschema.KymaConfigInput) (string, error) {
+	for _, c := range in.Components {
```

Comment:
you change that so do we have a uni test for that? Because I see that contract was changed and unit tests were not adjusted which is quite odd

Reply:
contract was not changed, but the multiline string was not tested, I've added a test